src/Microsoft.ML.DataView/VBuffer.cs

```diff
@@ -11,47 +11,75 @@
-        /// The number of items explicitly represented. This is == Length when the representation
-        /// is dense and &lt; Length when sparse.
+        /// The number of items explicitly represented. This equals <see cref="Length"/> when the representation
+        /// is dense and less than <see cref="Length"/> when sparse.
         /// </summary>
         private readonly int _count;
 
         /// <summary>
         /// The logical length of the buffer.
         /// </summary>
+        /// <remarks>
+        /// Note that if this vector <see cref="IsDense"/>, then this will be the same as the <see cref="ReadOnlySpan{T}.Length"/>
+        /// as returned from <see cref="GetValues"/>, since all values are explicitly represented in a dense representation. If
+        /// this is a sparse representation, then that <see cref="ReadOnlySpan{T}.Length"/> will be somewhat shorter, as this
+        /// field contains the number of both explicit and implicit entries.
+        /// </remarks>
         public readonly int Length;
 
         /// <summary>
-        /// The explicitly represented values.
+        /// The explicitly represented values. When this <see cref="IsDense"/>, the <see cref="ReadOnlySpan{T}.Length"/>
+        /// of the returned value will equal <see cref="Length"/>, and otherwise will have length less than
+        /// <see cref="Length"/>.
         /// </summary>
         public ReadOnlySpan<T> GetValues() => _values.AsSpan(0, _count);
 
         /// <summary>
-        /// The indices. For a dense representation, this array is not used. For a sparse representation
-        /// it is parallel to values and specifies the logical indices for the corresponding values.
+        /// The indices. For a dense representation, this array is not used, and will return the default "empty" span.
+        /// For a sparse representation it is parallel to that returned from <see cref="GetValues"/> and specifies the
+        /// logical indices for the corresponding values, in increasing order, between 0 inclusive and
+        /// <see cref="Length"/> exclusive, corresponding to all explicitly defined values. All values at unspecified
+        /// indices should be treated as being implicitly defined with the default value of <typeparamref name="T"/>.
         /// </summary>
         /// <remarks>
-        /// For example, if GetIndices() returns [3, 5] and GetValues() produces [98, 76], this VBuffer
-        /// stands for a vector with:
-        ///  - non-zeros values 98 and 76 respectively at the 4th and 6th coordinates
-        ///  - zeros at all other coordinates
+        /// To give one example, if <see cref="GetIndices"/> returns [3, 5] and <see cref="GetValues"/>() produces [98, 76],
+        /// this <see cref="VBuffer{T}"/> stands for a vector with non-zeros values 98 and 76 respectively at the 4th and 6th
+        /// coordinates, and zeros at all other coordinates. (Zero, because that is the default value for all .NET numeric
```

Comment:
I think the two `coordinate` usages should be changed to `indices`. We use the term "Index" and "Indices" everywhere else. Calling it "coordinate" here is confusing IMO.

Reply:
Yes, I agree, thanks Eric